cpp/src/traversal/bfs_kernels.cuh

```diff
@@ -889,7 +902,7 @@ __global__ void topdown_expand_kernel(
 
           int is_visited = vec_v_visited_bmap[iv] & m;
 
-          if (is_visited) vec_frontier_candidate[iv] = -1;
+          if (is_visited) vec_frontier_candidate[iv] = ~IndexType(0);
```

Comment:
Search for ~IndexType(0) and replace with invalid_vertex_id<IndexType>::value

Reply:
Done.